### ISSUE TITLE
Add `contains()` method to `Objectionary` and `OyLambda`

### ISSUE THREAD

**Issue by MikhailLipanin** [objectionary/eo#1566]
resolving [this PR](https://github.com/objectionary/eo/pull/1473) need to split it into several PR-s:

- `Objectionary` interface now has one more method `contains`, which checks the presence of object inside `Objectionary`
- `OyLambda` -- new way to create "fast" `Objectionary` in tests (it accepts `UncheckedFunc`, which represents work of `Objectionary.get()` method earlier)

**Comment by yegor256** [comment]
@MikhailLipanin I don't understand this ticket. What is broken? What is suggested?

**Comment by MikhailLipanin** [comment]
@yegor256, it is just a part of a PR that introduces `ProbeMojo`. Because he got big and hard to review, I splitted it to a several PR-s.

Explanation:
`ProbeMojo` should "Go through all `probe` metas in XMIR files, try to locate the objects pointed by `probe` in Objectionary and if found register them in catalog."

So, firstly, we should check, if the object is in `Objectionary`. The same approach is used in `PullMojo` [here](https://github.com/objectionary/eo/blob/master/eo-maven-plugin/src/main/java/org/eolang/maven/PullMojo.java#L133-L154). Sequentially, we need to reuse this code. For this one, I created `ChResolve` class, which, depending on attributes, creates  the desired `CommitHash`.

Secondly, to check if the object is in `Objectionary` (only its presence), we can make it by two ways:
- By trying to load object from `Objectionary` via `get()` method and if not succeed, try to catch an Exception and continue;
- By adding method `contains()` to `Objectionary` and implement it on each `Objectionary`-classes.

I have done the second one. But there is a trouble --- `Objectionary` earlier was a functional interface, but now it has 2 methods.
So, to use it in test a short notation for making an `Objectionary`, I created a new `OyFake` class. It helps to use it in tests.
